hack/generator/pkg/config/configuration.go

```diff
@@ -57,6 +57,12 @@ type Configuration struct {
 	// after init TypeTransformers is split into property and non-property transformers
 	typeTransformers     []*TypeTransformer
 	propertyTransformers []*TypeTransformer
+
+	// typesOutputModuleSuffix is the suffix to append to GoModulePath to build the actual go import path
+	// to include types with. For example if GoModulePath is "github.com/Azure/azure-service-operator/v2"
+	// and we locally put the generated types into an "api" folder then this suffix would be "api" so that
+	// when joined with GoModulePath the result is github.com/Azure/azure-service-operator/v2/api
+	typesOutputModuleSuffix string
```

Comment:
this was added but not actually used in the config file?

Reply:
It's not for the config file - it's cached and set during `initialize` (mostly to avoid having to return err or panic where I need it)